apps/src/templates/sectionsRefresh/CoteacherSettings.jsx

```diff
@@ -20,46 +19,47 @@ export default function CoteacherSettings({
 }) {
   const [inputValue, setInputValue] = useState('');
   const [addError, setAddError] = useState('');
-  const [removedCoteacherIds, setRemovedCoteacherIds] = useState([]);
-  const [coteacherToRemove, setCoteacherToRemove] = useState({});
+  const [coteacherToRemove, setCoteacherToRemove] = useState(null);
+
+  const initialCoteachers = React.useMemo(() => {
```

Comment:
This does look nicer!

Is there any concern about this component being rendered before the `sectionInstructors` are loaded? I'm mostly convinced this won't happen, but if it does the initialization step here will miss out.